jsapp/js/utils.ts

```diff
@@ -249,7 +249,7 @@ export function escapeHtml(str: string): string {
   return div.innerHTML;
 }
 
-export function renderCheckbox(id: string, label: string, isImportant: boolean) {
+export function renderCheckbox(id: string, label: string, isImportant?: boolean) {
```

Comment:
`function renderCheckbox(id: string, label: string, isImportant = false)` better reflects the intended usage. We seem to only every ignore this param, or set to true. Undefined essentially implies false in how we use it.